diff --git a/node_modules/vite/dist/node/utils/cssUtils.js b/node_modules/vite/dist/node/utils/cssUtils.js
index 1d5d775..610fd7b 100644
--- a/node_modules/vite/dist/node/utils/cssUtils.js
+++ b/node_modules/vite/dist/node/utils/cssUtils.js
@@ -74,7 +74,7 @@ async function compileCss(root, publicPath, { source, filename, scoped, vars, mo
         }
     }
     return await compileStyleAsync({
-        source,
+        source: preprocessOptions.prependData ? `${preprocessOptions.prependData}\n${source}` : source,
         filename,
         id: `data-v-${id}`,
         scoped,
